metricbeat/module/apache/status/status_test.go

```diff
@@ -35,6 +35,8 @@ import (
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 
 	"github.com/stretchr/testify/assert"
+
+	_ "github.com/elastic/beats/metricbeat/module/apache"
```

Comment:
Why does the module need to be included? Does otherwise some functionality miss?

Reply:
This is needed for the fields checks, without this the module fields are not included and then the check fails. The `TestAll` version includes all modules when including `github.com/elastic/beats/metricbeat/include`.

I was tempted of just moving this include to `mb/testing`, but this way we check that each module defines its own fields.

Common fields are included from `mb/testing`.

Not sure if there is other way of including the module fields only.